sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Should there be some content within this section of exploring the data exploration notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Not really other than to let the readers explore for themselves. Rewrote it slightly though so it's more clear.